db/migrations/20230104042228_create_plantsimg_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE plantsimg (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  imgurl VARCHAR(200) NOT NULL,
```

Comment:
```suggestion
  img_url VARCHAR(200) NOT NULL,
```
img_url이 200으로 부족할 것 같다고 리뷰드렸던 기억이 있습니다. 반영해주세요